app/views/course/assessment/answer/_worksheet.html.slim

```diff
@@ -1,16 +1,22 @@
 - if answer.submission.attempting? && answer.specific.is_a?(Course::Assessment::Answer::Programming)
+  - programming_answer = answer.specific
+  - if answer.question.actable.attempt_limit
+    - button_text = t('.run_code_with_limit', limit: programming_answer.attempting_times_left)
+  - else
+    - button_text = t('.run_code')
```

Comment:
How about add a hint in question creation page ? says that limits do not apply to staff.  

The other way I can think of is to put a tooltip on the button. Tooltip is useful for the first time, but I think for the rest of times it's annoying...